models/users-sequelize.mjs

```diff
@@ -0,0 +1,90 @@
+import { User, AbstractUsersStore } from './user.mjs';
+import { SQBook } from './books-sequelize.mjs';
+import Sequelize from 'sequelize';
+import {
+  connectDB as connectSequlz,
+  close as closeSequlz
+} from './index.mjs';
+import DBG from 'debug';
+const debug = DBG('users:users-sequelize');
```

Comment:
empty line between import and const, I'm not sure in this case

Reply:
Thanks :D, probably also applies